src/Math.inl

```diff
@@ -7,6 +7,18 @@ Vector
                   py::arg("y"),
                   py::arg("z"))
     .def(py::init<const MVector &>(), py::arg("src"))
+    .def(py::init([](const py::sequence & seq) {
+        if (seq.size() < 2 || seq.size() > 3)
+        {
+            throw std::invalid_argument("You must provide a list of 2 or 3 floats.");
+        }
+        double tmp[3] = {0.0, 0.0, 0.0};
+        std::transform(std::begin(seq), std::end(seq), std::begin(tmp),
+                [](pybind11::handle handle) -> double { return handle.cast<double>(); });
+        
+        return std::unique_ptr<MVector>(new MVector(tmp[0], tmp[1], tmp[2]));
+    }), "Create a new vector from a seqence or 2 or 3 floats")
```

Comment:
And you need `py::arg` here as well. Again, I've kept the same argument name as in maya's doc.
```suggestion
    }), py::arg("d"), "Create a new vector from a seqence or 2 or 3 floats")
```